cg/cli/set/case.py

```diff
@@ -77,18 +80,25 @@ def set_case(
     if panel_abbreviations:
         update_panels(case=case, panel_abbreviations=panel_abbreviations, status_db=status_db)
 
-    if priority:
+    if isinstance(priority, Priority):
```

Comment:
I would probably keep this check and change the check in abort on empty options to if len(options) == 0:. Would those changes miss anything?

Reply:
We could probably do an `if priority is not None` here as well, if that is easier to read